src/_posts/2016-11-07-aws-elastic-beanstalk.md

```diff
@@ -0,0 +1,58 @@
+---
+title: AWS Elastic Beanstalk
+---
+Appveyor does not support AWS Elastic Beanstalk deployment out of the box right now. However it can be automated in Appveyor with help of some scripting. Here is small guild based on [this support forum discussion] (<https://github.com/appveyor/ci/issues/45#issuecomment-165571187>).
```

Comment:
Some notes:

1. Grammar: `guilD`
2. Link is not displayed as link. There should not be space between `]` and `(`